.github/workflows/testpr.yml

```diff
@@ -35,6 +37,7 @@ jobs:
       with:
         environments: beta
         frozen: true
+        pixi-version: v0.40.2
```

Comment:
There is a specific reason to use this pixi version or it is just to increase reproducibility? If it is to increase reproducibility, could we pin to the latest version?